dia/.metadata/dia.manual.cs

```diff
@@ -0,0 +1,52 @@
+using Windows.Win32.Foundation;
+
+namespace Microsoft.Dia
+{
+    public enum SymTag
```

Comment:
Still seems problematic that we need this.

Reply:
Yeah I hear ya. Here's a bit more context. https://github.com/microsoft/win32metadata/discussions/745.